beacon-chain/types/block.go

```diff
@@ -27,6 +27,10 @@ type Block struct {
 	data *pb.BeaconBlock
 }
 
+type chainSearchService interface {
```

Comment:
Optional change, `chainService` sounds better to me and it's consistent with block chain and sync services.

Reply:
We already have `ChainService` (capital C) so I wanted to distinguish that this one is the one in block.go and is really for testing purposes only.  Do you still think we should change it? If so, i'd be glad to change it

Reply:
That's fine. I like chainSearchService as well